.github/workflows/canary-release.yml

```diff
@@ -40,10 +40,11 @@ jobs:
 
       - name: Publish Canary Packages
         id: changesets
-        uses: changesets/action@master  # TSCCR: no entry for repository "changesets/action"
+        uses: changesets/action@d89c1de63c7f28ac47ec85ed395f5f1d045d4697 # TSCCR: no entry for repository "changesets/action"
         with:
           # This expects you to have a script called release which does a build for your packages and calls changeset publish
           publish: npm run release:canary
+          createGithubReleases: false
```

Comment:
This is a handy input I found for the action, it should reduce GH release spam for canary releases 👀